apps/condo/domains/miniapp/utils/b2bAppServiceUserAccess/schema.utils.js

```diff
@@ -64,28 +73,45 @@ const getSchemaDocForReadOnlyPermissionField = (permissionFieldName) => {
  *
  * Overrides the default access behavior for the specified schema
  *
- * @typedef {Object} B2bAppServiceUserAccessSchemaConfig
+ * @typedef {Object} B2bAppServiceUserAccessListSchemaConfig
  * @property {Array.<string>} pathToOrganizationId - Way to get the organization id (default value: ['organization', 'id'])
  * @property {boolean} canBeRead - Service users can read schema (default value: true)
  * @property {boolean} canBeManaged - Service users can manage schema (default value: true)
  */
 
+/**
+ *
+ * Overrides the default access behavior for the specified service
+ *
+ * @typedef {Object} B2bAppServiceUserAccessServiceSchemaConfig
+ * @property {Array.<string>} pathToOrganizationId - Way to get the organization id (default value: ['organization', 'id'])
+ * @property {boolean} canBeExecuted - Service users can read schema (default value: true)
+ */
+
 /**
  *
  * Determines which models can be accessed by a service user linked to a B2B app
  *
- * @example Config for Organization schema
+ * @example Config for Organization schema and for some abstract service
  * {
- *    Organization: {
- *       // Default value ['organization', 'id'] => get value from <SchemaName>.organization.id
- *       // But for the Organization schema get value from <SchemaName>.id
- *       pathToOrganizationId: ['id'],
- *       // By default schemas can be manage, but for Organization schema cannot be managed
- *       canBeManaged: false,
+ *    lists: {
+ *        Organization: {
+ *            // Default value ['organization', 'id'] => get value from <SchemaName>.organization.id
+ *            // But for the Organization schema get value from <SchemaName>.id
+ *            pathToOrganizationId: ['id'],
+ *            // By default schemas can be manage, but for Organization schema cannot be managed
+ *            canBeManaged: false,
+ *         },
+ *    },
+ *    services: {
+ *        registerMeters: {
+ *            // All services can be executed by default, but you may change this behavior with next field:
+ *            canBeExecuted: false,
```

Comment:
What is this flag for? If you want to prohibit the execution of this service, then you simply do not need to add it to the configuration, do you? :)

For example, in the `UserRightsSet` configs we used just string (service name)